src/sbb.py

```diff
@@ -209,4 +281,6 @@ def get_sbb_contents(self) -> SBBContents:
                 # raise Exception('Unexpected SBB content')
                 pass
 
+        # TODO: Clean this up
+        #sbb_contents.t_values = self.t_values
```

Comment:
You don't assign anything to `self.t_values`, is this just dead code?

Reply:
Yep, same as above, just missed the comment